src/xdocs/config_filters.xml

```diff
@@ -2232,6 +2232,261 @@ public class UserService {
       </subsection>
     </section>
 
+    <section name="SuppressWithNearbyTextFilter">
+      <p>Since Checkstyle 10.8.2</p>
+      <subsection name="Description" id="SuppressWithNearbyTextFilter_Description">
+        <p>
+          Filter <code>SuppressWithNearbyTextFilter</code> uses plain text to suppress
+          nearby audit events. The filter can suppress all checks which have Checker
+          as a parent module.
+        </p>
+        <p>
+          Rationale: Sometimes there are legitimate reasons for suppressing a check.
+          When this is a matter of the code in question and not personal preference,
+          the best place to override the policy is in the code itself. Semi-structured
+          comments can be associated with the check. This is sometimes superior to
```

Comment:
>Semi-structured comments can be associated with the check.

I do not understand this, please rephrase.

Reply:
Removed.